packages/components/src/base-control/index.js

```diff
@@ -41,15 +34,6 @@ function BaseControl( { as = 'div', id, label, hideLabelFromVision, help, classN
 	);
 }
 
-BaseControl.VisualLabel = ( { as = 'span', className, children } ) => {
-	const Component = as;
-	const classes = classnames( 'components-base-control__label', className );
-
-	return (
-		<Component className={ classes }>
-			{ children }
-		</Component>
-	);
-};
+BaseControl.VisualLabel = VisualLabel;
```

Comment:
Should this be just an internal component not exposed on the @wordpress/components public API?

Reply:
@youknowriad It seems like this component is used in a couple of places (`block-library/src/video` and `components/gradient-picker`). It's not very likely that folks outside of Gutenberg are using it for their custom UIs, but we can't be certain.

Because this sub-component was already exposed, I left it the way it was.

Based on the `VisualLabel`'s markup, it doesn't seem like it would make sense outside the context of `@wordpress/components`.

If you think it's a good idea, I can refactor things to remove the static property from `BaseControl`

Reply:
> Because this sub-component was already exposed, I left it the way it was.

I missed that it was already exposed. Nvm my comment. Removing it will be a breaking change so let's keep it I guess.